ATC/src/ATC.cpp

```diff
@@ -85,6 +86,37 @@ void planeDemo() {
 	pthread_join(tid, NULL);
 }
 
+void OperatorConsoleDemo()
+{
+	OperatorConsole oc;
+	pthread_t tid;
+	pthread_create(&tid, NULL, &OperatorConsole::start, &oc);
+
+	while (oc.getChid() == -1)
+		;
+	int coid = ConnectAttach(0,0,oc.getChid(),_NTO_SIDE_CHANNEL,0);
+
+	int64_t sleepUntil;
+	for (int i = 0; i < 3; i++) {
+		sleepUntil = now() + 10L * 1000L * 1000L * 1000L;
+		while (now() < sleepUntil)
```

Comment:
Question here, what's the reason behind this loop sleep as compared to using
`std::this_thread::sleep_for(std::chrono::milliseconds(3 * 1000));` for example? Is it the need to have a higher resolution?

Reply:
To be honest, that's a better solution. I was just using this as a hacky busy-wait for demo purposes. Ideally we won't have to use any sort of sleep in the final code once we replace the demo code.